_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -276,11 +276,11 @@ public static function wrap_ui( $callback, $args = array() ) {
 		);
 		$args              = wp_parse_args( $args, $defaults );
 		$jetpack_admin_url = admin_url( 'admin.php?page=jetpack' );
-		$jetpack_about_url = ( Jetpack::is_active() || Jetpack::is_development_mode() )
+		$jetpack_about_url = ( Jetpack::is_active() || Jetpack::is_offline_mode() )
 			? admin_url( 'admin.php?page=jetpack_about' )
 			: Redirect::get_url( 'jetpack' );
 
-		$jetpack_privacy_url = ( Jetpack::is_active() || Jetpack::is_development_mode() )
+		$jetpack_privacy_url = ( Jetpack::is_active() || Jetpack::is_offline_mode() )
```

Comment:
Maybe we should switch to using the Status package here; there is no `is_offline_mode` method in the Jetpack class right now so this will create a Fatal error.

Reply:
48b817a4a . Thanks for catching that.